reStream.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 
-# these are probably the only two parameters you need to change
-ssh_host="root@10.11.99.1" # location of the remarkable
+# these are probably the only parameters you need to change
+ssh_host_usb="root@10.11.99.1" # location of the remarkable using usb
+ssh_host_wifi=""               # location of the remarkable using wifi
```

Comment:
It seems simpler to me to have an `-h --host` option which allows you to specify a different host than the default one. In that case the script stays simple (the nested `if`s are a bit convoluted) and the user can actively decide to connect trough WiFi or USB.

But since we then have multiple arguments, we should do some argument parsing. Are you familiar with using `getopt`?